warehouse/router/upload_stats.go

```diff
@@ -115,24 +126,6 @@ func (job *UploadJob) recordTableLoad(tableName string, numEvents int64) {
 		Name:  "tableName",
 		Value: capturedTableName,
 	}).Count(int(numEvents))
-
-	// Delay for the oldest event in the batch
-	firstEventAt, err := job.stagingFileRepo.FirstEventForUpload(job.ctx, job.upload)
-	if err != nil {
-		job.logger.Errorf("[WH]: Failed to generate delay metrics: %s, Err: %v", job.warehouse.Identifier, err)
-		return
-	}
-	if !job.upload.Retried {
-		syncFrequency := "1440"
-		if frequency := job.warehouse.GetStringDestinationConfig(job.conf, model.SyncFrequencySetting); frequency != "" {
-			syncFrequency = frequency
-		}
-		job.timerStat("event_delivery_time",
-			warehouseutils.Tag{Name: "tableName", Value: capturedTableName},
-			warehouseutils.Tag{Name: "syncFrequency", Value: syncFrequency},
-			warehouseutils.Tag{Name: "sourceCategory", Value: job.warehouse.Source.SourceDefinition.Category},
-		).Since(firstEventAt)
-	}
```

Comment:
Looks like we were not capturing this metric for retried uploads and we don't have that condition now. Why was this @achettyiitr @itsmihir

Reply:
It was mostly because when retries it would mess up the metrics because those will be days later.

Reply:
@achettyiitr can confirm, but I don't think of a reason why we can't capture the retried uploads.